source/linux/index_Edge_AI.rst

```diff
@@ -12,14 +10,18 @@ Edge AI
 
    For more information on edge AI software stack, refer `Edge AI Documentation <https://software-dl.ti.com/processor-sdk-linux/esd/AM62AX/10_01_00/exports/edgeai-docs/common/sdk_overview.html#>`_
 
-.. ifconfig:: CONFIG_part_variant in ('J721E')
+.. ifconfig:: CONFIG_part_variant in ('TDA4VM')
```

Comment:
@sinha-shreyash 
This is not right. There is not CONFIG_part_variant TDA4VM
TDA4VM uses 'J721E' as CONFIG_part_variant and 'edgeai' as CONFIG_image_type

Same comment below in the rest of the file

Reply:
updated